arches/install/arches-templates/project_name/templates/email/general_notification.htm

```diff
@@ -0,0 +1,30 @@
+{% load static %}
+{% load template_tags %}
+{% load i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        {% include 'custom_email_css.htm' %}
+    </head>
+
+    <body>
+
+        {% include 'custom_email_header.htm' %}
+
+        {% if username != "" %}
+            <p>Dear {{username}},</p>
+        {% endif %}
```

Comment:
Since we're parameterizing everything else in the email, should we go ahead and parameterize this as well?

Reply:
Could you clarify what you're asking for here?  Is it the use of 'Dear' as a static text in the template?

Reply:
Correct, all other text is parameterized. I could imagine someone may want to use another salutation besides `Dear`